src/main/java/org/team1540/robot2024/subsystems/indexer/IndexerIOSim.java

```diff
@@ -0,0 +1,66 @@
+package org.team1540.robot2024.subsystems.indexer;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.simulation.DCMotorSim;
+import edu.wpi.first.wpilibj.simulation.SimDeviceSim;
+import org.team1540.robot2024.Constants;
+
+import static org.team1540.robot2024.Constants.Indexer.*;
+
+public class IndexerIOSim implements IndexerIO {
+
+
+    private final DCMotorSim intakeSim = new DCMotorSim(DCMotor.getNEO(1), INTAKE_GEAR_RATIO,0.025);
+    private final DCMotorSim feederSim = new DCMotorSim(DCMotor.getNEO(1), FEEDER_GEAR_RATIO,0.025);
+//    private final SimDeviceSim beamBreakSim = new SimDeviceSim("Indexer Beam Break");
+    private final PIDController feederSimPID = new PIDController(FEEDER_KP, FEEDER_KI,FEEDER_KD);
+    private boolean isClosedLoop = true;
+    private double intakeVoltage = 0.0;
+    private double feederVoltage = 0.0;
+    private double feederVelocityRPS = 0.0;
```

Comment:
This should probably called `feederSetpointRPS`, `feederVelocityRPS` implies that this is the actual feeder velocity, which this is not.